srcpkgs/hplip-plugin/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'hplip-plugin'
+# Must be in sync with main hplip package
```

Comment:
include this notice in hplip too, otherwise someone will update only hplip and won't notice there is hplip-plugin